### PR TITLE
i18n(fr): Update `index.mdx` & `frontmatter.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Update `index.mdx` & `frontmatter.mdx` based on this commit 7bf0f20cdc8403e79220c4b8c5d3e42fb0d1672b

### CODE REVIEW

File: docs/src/content/docs/fr/index.mdx [hunk 1]
```diff
@@ -2,6 +2,15 @@
 title: Starlight 🌟 Construire des sites de documentation avec Astro
 description: Starlight vous aide à créer de magnifiques sites web de documentation très performants avec Astro.
 template: splash
+banner:
+  content: |
+    <span class="ph-banner">
+      <span class="ph-houston"><img src="/ph-houston.svg" alt="" width="72" height="72"/></span>
+      <span class="ph-text">
+        <strong>Welcome Product Hunters&nbsp;👋</strong>
+        <a href="https://www.producthunt.com/posts/starlight-by-astro">We are live right now!</a>
```
Comment (ElianCodes) [inline docs/src/content/docs/fr/index.mdx:11]:
```suggestion
        <strong>Bienvenue Product Hunters&nbsp;👋</strong>
        <a href="https://www.producthunt.com/posts/starlight-by-astro">Nous sommes en direct!</a>
```

**Review by ElianCodes — COMMENTED** [review]
I think this should be a correct French translation for the banner too?

**Comment by dreyfus92** [comment]
Thank you for reviewing this PR @ElianCodes 😁

**Review by delucis — APPROVED** [review]
Thanks team 🚀

**Pull request merged**